stack-graphs/src/c.rs

```diff
@@ -1471,6 +1495,7 @@ pub extern "C" fn sg_forward_partial_path_stitcher_from_partial_paths(
     partials: *mut sg_partial_path_arena,
     count: usize,
     initial_partial_paths: *const sg_partial_path,
+    config: sg_stitcher_config,
```

Comment:
...but maybe counter-intuitively, _here_ I think you should pass in the config by pointer!  (We're not currently worrying about [versioning our C ABIs](https://dcreager.net/2017/10/shared-library-versions/), but if we were to do that, passing by pointer means that the ABI calling convention for this function would not change even if we were to add more fields to the config struct.)